ArduPlane/mode_takeoff.cpp

```diff
@@ -150,6 +151,10 @@ void ModeTakeoff::update()
         plane.calc_nav_roll();
         plane.calc_nav_pitch();
         plane.calc_throttle();
+        //check if in long failsafe, if it is recall long failsafe now to get fs action via events call
+        if (plane.in_long_failsafe()) {
+        plane.failsafe_long_on_event(FAILSAFE_LONG, ModeReason::MODE_TAKEOFF_FAILSAFE);
```

Comment:
wont this repeated calling result in spamming the msg?`gcs().send_text(MAV_SEVERITY_WARNING, "%s Failsafe On: %s", (reason == ModeReason:: GCS_FAILSAFE) ? "GCS" : "RC Long", control_mode->name());`